### ISSUE TITLE
[Spell] Empowered Fire affecting Frostire Bolt DOT

### ISSUE THREAD

**Issue by cathyr1** [WoWManiaUK/Redemption#3488]
This issue is not about the entire spell http://www.wow-mania.com/armory/?spell=47610, but only about the damage over time effect, cause it is not working correctly.

**How this should work**
This damage effect is frostfire, so it benefits from both schools of magic, as well as it should benefit from your spell dmg.

**How it works**
Its damage benefits from spell damage and talents only when you have fire spec.
![fire](https://user-images.githubusercontent.com/30184491/63209336-bd703c00-c0df-11e9-8785-c5028b1e88c7.png)

If you have frost spec, its damage is basic, ignoring your spell power and talents.
![frost](https://user-images.githubusercontent.com/30184491/63209345-d11ba280-c0df-11e9-89e6-1404c8bb620c.png)

**Comment by Keader** [comment]
Talents can affect frostfire bolt as a fire and mage school.
Is clear that, fire spec has more talents that can affect DOTs.
(i dont even remember any frost spec talent, that add more damage in frostfire or even any frost spell lol)

**Comment by cathyr1** [comment]
You wanna see frost talents that affect damage of frostfire bolt? There are 3:
http://www.wow-mania.com/armory/?spell=12953
http://www.wow-mania.com/armory/?spell=31678
http://www.wow-mania.com/armory/?spell=44571

But mostly this is about spell damage. That dot is the same with 0 spell damage as it is with over 3 000 spell damage so this is surely not how it should work.

Please reopen and fix. Thanks.

**Comment by Keader** [comment]
@Dranah can you check this?

**Comment by Dranah** [comment]
There is a bug here, but not exactly what the report speaks of.
The spell doesn't have any base SP coefficient on its DOT part. The DOT deals base damage and that is intended.
The reason it begins scaling with SP in fire spec is that fire has a talent "Empowered Fire" which increases spell power benefit of, among others, FFB too, and this talent currently affects the DOT too. 

This is however unintended, that 15% boost should only apply to the direct damage of FFB as well.
TL;DR the DOT is a worthless debuff for pvp and no other purpose, it shouldn't scale with SP in anyway and that is blizzlike. The bug here is that the fire talent is currently boosting it, though it shouldn't.
It should do base damage with deep fire talents taken too, you can see this on retail logs:
http://www.worldoflogs.com/reports/rt-kmbaybav89c03kol/details/2/?s=11828&e=11958
And wowwiki also notes it has 0% coeff, and is unaffected by talent: https://wowwiki.fandom.com/wiki/Spell_power_coefficient

How it should work: "Empowered Fire" ID 31658 should not apply to the DoT ticks of FFB.
How it works atm: "Empowered Fire" is currently boosting the DOT ticks adding 15% sp coeff per tick.

**Comment by cathyr1** [comment]
The exact thing is for fireball dot http://www.wow-mania.com/armory/?spell=42833. It also benefits from spell power only with Empowered Fire talent. Both spells (dots) should have the same behavior.

**Comment by Dranah** [comment]
Then need to open separate thread for fireball to fix that one too. That being said, it holds less relevance as a fireball spec mage has it glyphed and thus no DOT effect on FB.

**Comment by cathyr1** [comment]
It is the same problem for both spells so one fix of Empowered Fire talent should fix both. It would be quite unlogic to fix (nerf) frostfire and let fireball boosted. That glyph reduces cast by only 0,15 secs but the improved dot is quite strong so many can choose dot and different glyph.

**Comment by Dranah** [comment]
Makes sense, although it's up to @Keader to decide if it can be solved with just one fix. Nonetheless, shouldn't be left broken.
